lib/ansible/modules/cloud/amazon/rds_instance.py

```diff
@@ -572,7 +582,7 @@ def modify_db_instance(module, conn):
 
     # guess that this changed the DB, need a way to check
     diff = existing_instance.diff(instance.data)
-    changed = not not diff
+    changed = not diff
```

Comment:
I was wondering, However the interesting thing was that this change fixed the integration test case which was meant to show change and which had a positive existing diff.